javascript/react/redux-i/your-first-reducer.md

```diff
@@ -0,0 +1,162 @@
+---
+author: kapnobatai136
+
+levels:
+  - advanced
+  - medium
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+
+aspects:
+  - introduction
+
+---
+# Your first reducer
+
+---
+## Content
+
+In a previous insight we have created a simple `reducer` that looked something like this:
```

Comment:
Why `reducer` and not reducer? It's not code, right?

Reply:
You are right, wanted to evidentiate it but I could've done that with bold. I've removed the code ticks from `reducer`.